### DISCUSSION TITLE
I have a problem [1.7.10]

### DISCUSSION THREAD

**Discussion by CreamyOmelet** [Tropicraft/Tropicraft#544] [Support]
I can't enter the tropical realm in multiplayer [LAN Server], but in singleplayer it works just fine. When I drink the Pina Colada it doesn't even load terrain, I just stay where I was, like I never left or done anything. We are using Tropicraft with other mods such as Legends mod, Twilight Forest, The Aether and Galacticraft with Extra Planets add-on. Please help me, I really want to explore the mod.

**Comment by jaskarth** [comment]
Do you have a log file that you could share? It's hard to diagnose the issue without one unfortunately.
